react/components/ProgressBar/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import Step from './Step'
+
+class ProgressBar extends PureComponent {
+  isFirstElement(index) {
+    return index === 0
+  }
+
+  isLastElement(index, array) {
+    return index === array.length - 1
+  }
+
+  render() {
+    const { steps, late } = this.props
+
+    const stepsElements = steps.map((type, index) => {
+      const isFirstElement = this.isFirstElement(index)
+      const isLastElement = this.isLastElement(index, steps)
+
+      const marginRightCSS = isLastElement ? '' : 'mr2'
+      return (
+        <div className={`w-100 ${marginRightCSS}`} key={index}>
+          <Step
+            type={type}
+            roundLeft={isFirstElement}
+            roundRight={isLastElement}
+            danger={late}
+          />
+        </div>
+      )
+    })
+    return <div className="w-100 mb2 inline-flex flex-row">{stepsElements}</div>
+  }
+}
+
+ProgressBar.propTypes = {
+  /** Array of steps, it should be composed of instances of the following strings:'completed', 'inProgress' and 'toDo' */
+  steps: PropTypes.arrayOf(PropTypes.oneOf(['completed', 'inProgress', 'toDo']))
+    .isRequired,
+  /** Boolean representing if the progress is late or not, if true this changes the color of the steps */
+  late: PropTypes.bool,
```

Comment:
This has a meaning that's too specific I think; I suggest just changing it to "danger" or "warning" or  "alert" or somesuch